### DISCUSSION TITLE
Syntax Error

### DISCUSSION THREAD

**Discussion by 0xkumar** [Cyfrin/foundry-full-course-cu#470] [Q&A] [answered]
I Tried to fetch the data from the array of struct but i got the following error
![Screenshot 2023-07-27 225921](https://github.com/Cyfrin/foundry-full-course-f23/assets/136639853/8a30515c-c80b-45c1-974f-aa987ee89de9)

The testing contract has a struct of type Transaction and the array of type Transaction named transactions. 

![Screenshot 2023-07-27 230114](https://github.com/Cyfrin/foundry-full-course-f23/assets/136639853/0e94e777-018d-49f6-90d6-f8f372ae3173)

**Comment by Niferu** [comment]


**Reply by robbiesumner** [reply]
`uint256` is the same type as `uint`, that is not the issue here, but I like the solution with the getter function

**Comment by robbiesumner** [comment]
You'll have to remove the `.value` after `wallet.transactions(0)`:
`wallet.transactions(0)` returns a tuple containing five different values. By writing `(,uint256 _value,,,)` you are already extracting that exact value from the tuple. No need to append `.tuple`.

**Reply by robbiesumner** [reply, marked as answer]
So that would be:
```solidity
(,uint256 _value,,,) = wallet.transactions(0); // without .value
```
